AndriyZhornyak/Eveything/HMW11/HMW11.py

```diff
@@ -0,0 +1,40 @@
+# Task1
+# try:
+#     ask = int(input())
+#     if ask <= 0:
+#         raise ValueError("Negative number") 
+#     if ask % 2 == 0:
+#         print("The number is even")
+#     else:
+#         print("The number is odd")
+# except ValueError:
+#     print(ValueError)
+# Task2
```

Comment:
def task2():
    try:
        ask = int(input("Enter a number from 1 to 7: "))
        if ask < 1 or ask > 7:
            raise ValueError("Input must be between 1 and 7.")
        days = {
            1: "Monday",
            2: "Tuesday",
            3: "Wednesday",
            4: "Thursday",
            5: "Friday",
            6: "Saturday",
            7: "Sunday"
        }
        print(days[ask])
    except ValueError as e:
        print(e)


task2()